setup.py

```diff
@@ -48,7 +48,7 @@
         'Programming Language :: Python :: Implementation :: PyPy'
     ],
 
-    install_requires=['coincurve>=4.3.0', 'requests'],
+    install_requires=['coincurve>=4.3.0', 'requests', 'pyaes'],
```

Comment:
It's very cool that pyaes has no dependencies, but it's a bit dated and possibly prone to timing attacks. I'd just like to point that out.

Is this what Electron Cash uses?

Reply:
Electron cash uses cryptodome; however, falls back to pyaes if cryptodome import fails.

Pyaes is simply a pythonic implementation of AES-128. I think, mitigating attack such as timing attack would be more relevant on how bitcash -- and apps/implementation that use bitcash -- uses pyaes.
So mostly, any implementation that uses bitcash's ECIES would need to mitigate timing attacks by choosing when it declares the message to be invalid, right when bitcash tells it that AES key/iv is bad, or when it further verifies the content of the "successfully" decrypted data to be bad.